#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_disabled_tests.dpatch by Norbert Tretkowski <norbert@tretkowski.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: As we completely disabled ndbcluster

@DPATCH@
diff -Nur mysql-dfsg-5.1-5.1.44.orig/mysql-test/suite/innodb/t/disabled.def mysql-dfsg-5.1-5.1.44/mysql-test/suite/innodb/t/disabled.def
--- mysql-dfsg-5.1-5.1.44.orig/mysql-test/suite/innodb/t/disabled.def	2010-02-04 13:09:18.000000000 +0100
+++ mysql-dfsg-5.1-5.1.44/mysql-test/suite/innodb/t/disabled.def	2010-02-21 20:44:22.857703756 +0100
@@ -1 +1,2 @@
 innodb-index: InnoDB: Error: table `test`.`t1#1` already exists in InnoDB internal
+innodb_information_schema: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=570693
